### PR TITLE
feat: ELECTRON-1204 - add support for ttl

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
## Description
Support TTL expiry check for test builds.
[ELECTRON-1204](https://perzoinc.atlassian.net/browse/ELECTRON-1204)

## Solution Approach
N/A

## Related PRs
N/A

Notes: Add support for checking expiry on builds with a pre-determined `time-to-live` value specified while creating a build.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +18,12 @@
     "packed-mac": "npm run unpacked-mac && packagesbuild -v installer/mac/symphony-mac-packager.pkgproj",
     "prebuild": "npm run compile && npm run rebuild && npm run browserify-preload",
     "rebuild": "electron-rebuild -f",
+    "setexpiry": "gulp setExpiry",
     "start": "npm run compile && npm run browserify-preload && cross-env ELECTRON_DEV=true electron .",
     "test": "npm run lint && cross-env ELECTRON_QA=true jest --config jest.config.json --runInBand",
     "unpacked-mac": "npm run prebuild && npm run test && build --mac --dir",
     "unpacked-win": "npm run prebuild && npm run test && build --win --x64 --dir",
-    "unpacked-win-x86": "npm run prebuild && npm run test && build --win --ia32 --dir"
+    "unpacked-win-x86": "npm run prebuild && npm run test && build --win --ia32 --dir",
```
Comment (KiranNiranjan) [inline package.json]: The comma is not allowed here.

Comment (VishwasShashidhar) [reply package.json]: Fixed.

**Pull request merged**